src/sentry/integrations/utils/code_mapping.py

```diff
@@ -61,15 +61,19 @@ class UnsupportedFrameFilename(Exception):
 class FrameFilename:
     def __init__(self, frame_file_path: str) -> None:
         self.raw_path = frame_file_path
-        if frame_file_path[0] == "/":
+        is_windows_path = False
+        if "\\" in frame_file_path:
```

Comment:
can this conflict with the escape character on Linux? e.g: `\` as an escape for a space in the path? or is that just a shell thing?